web/admin.py

```
@@ -101,14 +101,32 @@ class WaitingRoomAdmin(admin.ModelAdmin):
+        "user", 
+        "is_teacher", 
+        "points",  # NEW: Add points to list view
+        "expertise", 
+        "created_at", 
+        "updated_at"
+    )
+    list_filter = (
+        "is_teacher", 
+        "points",  # NEW: Add to filters
+        "created_at", 
+        "updated_at"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)